acceptance/acceptance_test.go

```diff
@@ -92,7 +91,7 @@ func TestInprocessMode(t *testing.T) {
 	require.Equal(t, 1, testAccept(t, true, "selftest/server"))
 }
 
-func testAccept(t *testing.T, InprocessMode bool, singleTest string) int {
+func testAccept(t *testing.T, inprocessMode bool, singleTest string) int {
```

Comment:
nit: I agree it should be lowercased, but it's unrelated. In that case I'd suggest a separate PR with just a rename to keep your main PR smaller.